cachefactory/cachefactory.d.ts

```diff
@@ -0,0 +1,403 @@
+// Type definitions for CacheFactory 1.4.0
+// Project: https://github.com/jmdobry/CacheFactory
+// Definitions by: Vaggelis Mparmpas <https://github.com/vag1830>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module CacheFactory {
+	export interface IStoreImplementation {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions
> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.